geoapps_utils/application/dash_application.py

```diff
@@ -62,7 +63,11 @@ def __init__(
             # Params for initialization are coming from params
             # ui_json_data is provided
             self.params = params
-        elif ui_json is not None and Path(ui_json.path).exists():
+        elif (
+            ui_json is not None
+            and getattr(ui_json, "path", None) is not None
+            and Path(ui_json.path).exists()  # type: ignore
```

Comment:
This is the error I get without the type ignore. I added the get for ui_json.path being None, but maybe it doesn't recognize getattr?

![mypy](https://github.com/MiraGeoscience/geoapps-utils/assets/105238245/9ca8089c-cf39-41ad-90e7-3964516b30d6)

Reply:
see how it can be all addressed in https://github.com/MiraGeoscience/geoapps-utils/pull/12
But that is beyond scope for this issue. Good to go as is here